src/software/simulation/er_force_simulator.cpp

```diff
@@ -371,15 +400,68 @@ SSLSimulationProto::RobotControl ErForceSimulator::updateSimulatorRobots(
     return robot_control;
 }
 
+
+// Takes in current velocity and angular velocity and a target Direct Control primitive
+// converts current and target velocities to Wheel velocities
+// ramps the target primitive based on the current velocities
+// and returns a pointer to a new Direct Control primitive
```

Comment:
nit: comments like this are more fitting in the header file, as part of the javadoc